### PR TITLE
Homepage is customized for batch 5

### PR DESCRIPTION
**Pull request by temitopeohassan**
## Description

_Concise description of proposed changes, We recommend using screenshots and videos for better description_

## Additional Information

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #6 _ 

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address: 0x2b2542b313385AF7453623765383CE3cc5D3c9cF

### CODE REVIEW

**Comment by technophile-04** [comment]
Hey thanks @temitopeohassan !! The hero section looks great !! 

Regarding builders section I think we can remove it since we already have it on https://batch5.buidlguidl.com/builders and seems a bit repetitive, what do you think ?

**Comment by temitopeohassan** [comment]
I've removed the builders list from the homepage as recommended

File: packages/nextjs/app/page.tsx [hunk 1]
```diff
@@ -14,20 +14,31 @@ const Home: NextPage = () => {
   return (
     <>
       <div className="flex items-center flex-col flex-grow pt-10">
```
[the remark this answers is not in the document]
Comment (temitopeohassan) [reply packages/nextjs/app/page.tsx]: I've adjusted the style as recommended

File: packages/nextjs/app/page.tsx [hunk 2]
```diff
@@ -14,20 +14,31 @@ const Home: NextPage = () => {
...
+                  <span>{(checkedInCount || 0n).toString()}</span>
+                ) : (
+                  <span className="loading loading-spinner loading-m"></span>
+                )}
+              </p>
+            </div>
+          </div>
         </div>
 
         <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
```
[the remark this answers is not in the document]
Comment (temitopeohassan) [reply packages/nextjs/app/page.tsx]: I've removed the section

**Review by technophile-04 — APPROVED** [review]
Tysm @temitopeohassan this looks great !!

**Pull request merged**